cypress/integration/tests/developer/controls/businessservices/crud.test.ts

```diff
@@ -53,7 +54,7 @@ describe("Business service CRUD operations", { tags: "@tier1" }, () => {
     it("Business service CRUD", function () {
         const businessService = new BusinessServices(data.getCompanyName(), data.getDescription());
 
-        selectUserPerspective("Developer");
+        selectUserPerspective(migration);
 
```

Comment:
Can we do BusinessServices.openList(); here ?

Reply:
Here it was just a single line to navigate to user perspective. Don't know why. Don't want to break the test so will investigate and fix here.